src/coordinator/ts/m3db/types.go

```diff
@@ -27,17 +27,20 @@ import (
 	"github.com/m3db/m3x/ident"
 )
 
-// SeriesBlock is a placeholder until it is added to M3DB
+// SeriesBlock contains the individual series iterators
 type SeriesBlock struct {
 	Start          time.Time
 	End            time.Time
 	SeriesIterator encoding.SeriesIterator
```

Comment:
Might be good to make the SeriesIterator private to ensure you can't do something weird like close them

Reply:
Won't we need to access them when we process the query later on?

Reply:
That's a problem for future us ;)

Would be good to ensure we don't accidentally close it if we do need to expose them; e.g. the SeriesBlock could just expose a `.Current() (ts.Datapoint, xtime.Unit, ts.Annotation)` and `.Next() bool` method which would defer to its seriesIterator